src/server/start.ts

```diff
@@ -25,9 +25,16 @@
 
 import { Server } from '@hapi/hapi'
 import Logger from '@mojaloop/central-services-logger'
+import { Util } from '@mojaloop/central-services-shared'
+import Config from '../shared/config'
+import Metrics from '@mojaloop/central-services-metrics'
 
 export default async function start (server: Server): Promise<Server> {
   await server.start()
   Logger.info(`thirdparty-api-adapter is running @ ${server.info.uri}`)
+  await Util.Endpoints.initializeCache(Config.ENDPOINT_CACHE_CONFIG)
```

Comment:
shouldn't it be done before `await server.start()` The cache is runtime dependency for server code...

Reply:
Ok